mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -176,30 +172,21 @@ def deploy(self,
         parameters['converter_type'] = 'DETECTION'
         parameters['model_parameters'] = self.inferencer.configuration
         parameters['model_parameters']['labels'] = LabelSchemaMapper.forward(self.task_environment.label_schema)
-        name_of_package = "demo_package"
+
         with tempfile.TemporaryDirectory() as tempdir:
-            copyfile(os.path.join(work_dir, "setup.py"), os.path.join(tempdir, "setup.py"))
-            copyfile(os.path.join(work_dir, "requirements.txt"), os.path.join(tempdir, "requirements.txt"))
-            copytree(os.path.join(work_dir, name_of_package), os.path.join(tempdir, name_of_package))
-            config_path = os.path.join(tempdir, name_of_package, "config.json")
-            with open(config_path, "w", encoding='utf-8') as f:
-                json.dump(parameters, f, ensure_ascii=False, indent=4)
-            # generate model.py
-            if (inspect.getmodule(self.inferencer.model) in
-               [module[1] for module in inspect.getmembers(model_wrappers, inspect.ismodule)]):
-                copyfile(model_file, os.path.join(tempdir, name_of_package, "model.py"))
-            # create wheel package
-            subprocess.run([sys.executable, os.path.join(tempdir, "setup.py"), 'bdist_wheel',
-                            '--dist-dir', tempdir, 'clean', '--all'])
-            wheel_file_name = [f for f in os.listdir(tempdir) if f.endswith('.whl')][0]
-
-            with ZipFile(os.path.join(tempdir, "openvino.zip"), 'w') as zip:
-                zip.writestr(os.path.join("model", "model.xml"), self.model.get_data("openvino.xml"))
-                zip.writestr(os.path.join("model", "model.bin"), self.model.get_data("openvino.bin"))
-                zip.write(os.path.join(tempdir, "requirements.txt"), os.path.join("python", "requirements.txt"))
-                zip.write(os.path.join(work_dir, "README.md"), os.path.join("python", "README.md"))
-                zip.write(os.path.join(work_dir, "demo.py"), os.path.join("python", "demo.py"))
-                zip.write(os.path.join(tempdir, wheel_file_name), os.path.join("python", wheel_file_name))
+            with ZipFile(os.path.join(tempdir, "openvino.zip"), 'w') as arch:
```

Comment:
Are the temporary directory and archive file needed at all, or having a file-like object is enough?